governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+# Interpretation Exercise 01
+
+---
+
+## Proposed Rewrite of GOV 12
+
+To redress the aforementioned concerns, I would rewrite GOV 12 to achieve three aims. 
+
+First, I create a general protocol for GOV 12 and restrict its application to "substantial threats" to the functioning of Maker Governance. This is a far higher showing to meet than the current one. It redresses the risk stemming from ambiguity/overbreadth, while preserving to a degree the original "bootstrapping" intention.
```

Comment:
This definitely reduces the overbreadth, but potentially does defeat the purpose of the articles. They are intended to be very broad, so that they can cover unanticipated consequences. Some consequences are still things you want to fix quickly, but wouldn't count as substantial threats.

I think in conclusion I question whether you need to be quite so restrictive here, especially given your second major change to require sign-off from relevant scope facilitators + advisors.

Third change does cover exceptions around launch, but I'm not sure that is broad enough to cover potential issues at this stage.

Reply:
One other danger here is the level of (perceived) complexity is higher. In an ideal world, I think we want people to be able to easily engage with the scopes. Everytime we add more words (especially long words), it makes that harder.